src/common/vitals/index.js

```diff
@@ -0,0 +1,8 @@
+import { cumulativeLayoutShift } from './cumulative-layout-shift'
+import { firstContentfulPaint } from './first-contentful-paint'
+import { firstInputDelay } from './first-input-delay'
+import { firstPaint } from './first-paint'
+import { interactionToNextPaint } from './interaction-to-next-paint'
+import { largestContentfulPaint } from './largest-contentful-paint'
+
+export { cumulativeLayoutShift, firstContentfulPaint, firstInputDelay, firstPaint, interactionToNextPaint, largestContentfulPaint }
```

Comment:
Could this whole file be written as `export * from 'blah'`

Reply:
Yes but this guards from adding stuff later to that file that may not be meant to be barrel'd up. 

I'm good either way tho

Reply:
True. Personally I hate barrels because I've been on the shit end of figuring out why bundles where so big and including stuff. I am curious if it matters in the size department of our outputs. Thats really all I care about here.

Reply:
Yeah true. Let's just skip the barrel and be explicit